cdap-docs/reference-manual/source/http-restful-api/http-restful-api-v3/preferences.rst

```diff
@@ -0,0 +1,163 @@
+.. meta::
+:author: Cask Data, Inc.
+    :description: HTTP RESTful Interface to the Cask Data Application Platform
+      :copyright: Copyright © 2015 Cask Data, Inc.
+
+.. _http-restful-api-preferences:
+.. _http-restful-api-v3-namespace:
+
+===========================================================
+Preferences HTTP RESTful API
+===========================================================
+
+.. highlight:: console
+
+Use the CDAP Preferences HTTP API to save, retrieve, delete preferences in CDAP.
+
+Preferences, their use and examples, are described in the :ref:`Admin' Manual: Preferences
+  <preferences>`.
```

Comment:
Preferences, their use and examples of using them, are described in the :ref:`Administration Manual: Preferences
 <preferences>`.